ads/adthrive.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adthrive(global, data) {
+  validateData(data, ['siteId', 'adUnit'], ['sizes']);
+  loadScript(global, 'https://ads.adthrive.com/sites/'
+      + encodeURIComponent(data.siteId) + '/amp.min.js');
```

Comment:
Do you mean 8 spaces then? Is there a linting test I'm missing that checks for this? I based my indentation on how these scripts handled it: adreactor.js, advertserve.js, and brainy.js.